### ISSUE TITLE
🚑🚨 Urgent Bug - Broken Livestream Banner API Routes

### ISSUE THREAD

**Issue by Harry-Ross** [SSWConsulting/SSW.Website#1235]
**Acceptance Criteria:**
* No more 500s

**Comment by bradystroud** [comment]
We discovered the problem was from updating the [msal-node](https://www.npmjs.com/package/@azure/msal-node) package. @Harry-Ross downgraded it in #1237 

The issue with the package has already been reported https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/6339
Once thats fixed the package can be updated again

**Comment by bradystroud** [comment]
This issue was fixed https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/6347

I updated the package in #1249

**Comment by bradystroud** [comment]
This problem is back after merging the above PR.

I have reverted my change in #1252 

We also discovered that some parts of App Insights are not working. 

https://github.com/SSWConsulting/SSW.Website/blob/c401a949dfc746e7fdc06108e83ea8c4d023999b/pages/api/get-livestream-banner.ts#L51-L55

We are logging exceptions but they are not appearing on App Insights. The failed request is appearing though.
